styx-common/src/main/java/com/spotify/styx/serialization/PersistentEvent.java

```diff
@@ -59,7 +59,7 @@
     @JsonSubTypes.Type(value = PersistentEvent.Submitted.class, name = "submitted")
     })
 @JsonInclude(Include.NON_ABSENT)
-class PersistentEvent {
+public class PersistentEvent {
```

Comment:
package accessibility fails serialisation in jackson 2.10.x. I haven't found the related changelog though.